City-of-Helsinki/drupal-module-helfi-api-base
# Link processor fixes

## How to install

- `composer require drupal/helfi_api_base:dev-UHF-X-test-fixes`
- `drush cr`

## How to test

- Add an "anchor" link (`#anchor`) to a textarea
- Open the page with anchor link and check that no `Failed to parse link: xxx` errors are logged (`docker compose logs app`)
